src/classNames.js

```diff
@@ -0,0 +1,15 @@
+import style from 'flexboxgrid';
+
+function defaultClassNameLookup(className) {
+  return style[className] || className;
+}
+
+let classNameLookup = defaultClassNameLookup;
+
+export function setClassNameLookup(fn) {
```

Comment:
Is this function still needed? I think that tests passing without using this function would prove that the automatic fallback to regular classes works.

Another reason why I don't want this function to exist is because we want only 2 options:

1. CSS Modules
2. classes which flexboxgrid expect

This function would allow users to map classes to something else as well, in which case styles wouldn't apply.